### PR TITLE
Remove unused config

### PR DESCRIPTION
**Pull request by richard-dds**
This PR cleans up a bit of configuration cruft. Let me know if there's anything that I missed!

### CODE REVIEW

File: .secrets.baseline [hunk 1]
```diff
@@ -198,7 +183,7 @@
       }
     ]
   },
-  "version": "0.13.0",
+  "version": "0.12.7",
```
Comment (graham-dds) [inline .secrets.baseline]: I don't know what this version key is controlled by, but is it bad that the value rolled back?

Comment (dandds) [reply .secrets.baseline]: Yeah, @richard-dds you might need to update your dev dependencies.

**Comment by richard-dds** [comment]
@dandds I removed `config/selenium.ini`, but I didn't see any storage variables except for `CRL_STORAGE_` and `AZURE_STORAGE`, which I believe are necessary. Do you know of any that I'm missing?

**Comment by dandds** [comment]
@richard-dds Whoops, looks like I had a bunch in a gitignored INI file in my copy of the repo. There is one rogue `STORAGE_PROVIDER` in config/test.ini.

**Pull request merged**